pages/common/clj.md

```diff
@@ -0,0 +1,29 @@
+# clj
+
+> Clojure tool to start a REPL or invoke a specific function with data.
+> All options can be defined in a `deps.edn` file.
+> More information: <https://clojure.org/guides/deps_and_cli>.
+
+- Start a REPL:
+
+`clj`
+
+- Exec function:
+
+`clj -X {{namespace/fn_name}}`
```

Comment:
Generally, we prefer to avoid shortening names to aid clarity :-)
```suggestion
`clj -X {{namespace/function_name}}`
```

Also, is there a long form of the argument there? Generally we prefer the long forms of arguments in tldr pages. The page goes for other examples.